src/components/button/button.tsx

```diff
@@ -25,17 +21,18 @@ export const Button: FC<ButtonProps> = ({
   onClick = () => {},
```

Comment:
Let's mark returning type as void, () => {} seems to be a bit misleading

Reply:
@Azein i think it is void, it's not type definition, it's the default value here